Tasks/Khatsuk/Task2_class.py

```diff
@@ -0,0 +1,67 @@
+from datetime import datetime
+class time_to_text:
```

Comment:
as far as I understood, you want the class in order to avoid the data structure creation inside the function, but there is a very simple way to avoid it and not misuse a classes functionality - just move the data structure out of the function/class

you created the class only to store some static data in it, this is not the best approach and not the main goal of the classes and OOP